OpenStack-accounting/usr/local/sbin/nova_extract_accounting.py

```diff
@@ -1,64 +1,25 @@
-#!/usr/bin/python
+#!/usr/bin/python3
+import accountinglib
+
+import sys
 import time
 import datetime
-#from datetime import datetime,time
-import json
-import requests
-import sys
-import sqlalchemy
-from sqlalchemy.sql import select
-from sqlalchemy.orm import sessionmaker
-from ConfigParser import SafeConfigParser
-
-#This config parsing needs rewriting to support python3
-#The parsing of the nova.conf seems to not work with the python3 version
 
-# Read from config file
-parser = SafeConfigParser()
-try:
-   parser.read('/etc/influxdb.conf')
-   host = parser.get('db', 'host')
-   database = parser.get('db', 'database')
-   username = parser.get('auth', 'username')
-   password = parser.get('auth', 'password')
-   instance = parser.get('cloud','instance')
-except:
-   raise RuntimeError('Unable to read from config file')
-   sys.exit(1)
-
-url = 'http://'+host+'/write?db='+database +'&precision=s'
 
 nowtime = time.localtime()
 
-try:
-   parser.read('/etc/nova/nova.conf')
-   connectionstring = parser.get('database','connection')
-except:
-   raise RuntimeError('Unable to read from config file')
-   sys.exit(1)
-
+print("Nova Accounting run start")
 starttime=sys.argv[1]
-print(starttime)
+print("Start Time = " + starttime)
 endtime=sys.argv[2]
```

Comment:
What is `sys.argv` ?

Reply:
In this case you only have 2 arguments so using `sys.argv` is fine 
Ideally using `argparse`  module you can write a more user-friendly interface - for example you specify a `--help` command.

I recommend if you're going to stick with `sys.argv` - you should raise an error if you don't receive both values and specify what the program wants. You may also want to check the inputs and make sure they are in the right format